onnxruntime/core/providers/cuda/activation/activations.cc

```diff
@@ -73,7 +73,9 @@ namespace cuda {
 UNARY_ACTIVATION_OP_HFD(Elu, 6);
 UNARY_ACTIVATION_OP_HFD(HardSigmoid, 6);
 UNARY_ACTIVATION_OP_HFD(LeakyRelu, 6);
-UNARY_ACTIVATION_OP_HFD(Relu, 13);
+UNARY_ACTIVATION_OP_HFD(Relu, 14);
+UNARY_ACTIVATION_OP_VERSIONED_HFD(Relu, 13, 13);
+UNARY_ACTIVATION_OP_VERSIONED_TYPED(Relu, 13, 13, BFloat16)
 UNARY_ACTIVATION_OP_VERSIONED_HFD(Relu, 6, 12);
```

Comment:
Seems error prone to have to add this separately. Should there be a new 'versioned' macro that's equivalent to UNARY_ACTIVATION_OP_HFD so it's easier to migrate something from 'latest' to 'versioned'?